### ISSUE TITLE
typo in the plan name for the database, should set to essential not esential

### ISSUE THREAD

**Issue by stephanerenou** [ovh/terraform-provider-ovh#288]
### Terraform Version
Terraform v1.2.7

### Affected Resource(s)
Please list the resources as a list, for example:
- postgres database documentation at: https://registry.terraform.io/providers/ovh/ovh/latest/docs/resources/cloud_project_database

### Terraform Configuration Files
```hcl
resource "ovh_cloud_project_database" "postgresql" {
  service_name = "8256983a724d4d5e987816e48c7430b9"
  description  = "my-first-postgresql"
  engine       = "postgresql"
  version      = "14"
  plan         = "esential"
  nodes {
    region     = "GRA"
  }
  flavor = "db1-4"
}
```

### Debug Output
│ Error: calling Post /cloud/project/8256983a724d4d5e987816e48c7430b9/database/postgresql with params my-first-postgresql: 14:
│        "Error 404: \"Plan esential does not exist\""
│ 
│   with ovh_cloud_project_database.postgresql,
│   on simple_instance.tf line 29, in resource "ovh_cloud_project_database" "postgresql":
│   29: resource "ovh_cloud_project_database" "postgresql" {

### Panic Output
If Terraform produced a panic, please provide a link to a GitHub Gist containing the output of the `crash.log`.

### Expected Behavior
update documentation to refer to essential and not esential

### Actual Behavior
What actually happened?

### Steps to Reproduce

1. `terraform apply`
produced above error

### Important Factoids
Are there anything atypical about your accounts that we should know? For example: Running in EC2 Classic? Custom version of OpenStack? Tight ACLs?

### References
Are there any other GitHub issues (open or closed) or Pull Requests that should be linked here? For example:
- GH-1234

**Comment by scraly** [comment]
Thanks for the issue,
it will be fixed in PR #290 :)

**Comment by scraly** [comment]
Fixed in 0.20
